gensim/models/wrappers/ldamallet.py

```diff
@@ -118,7 +126,16 @@ def __init__(self, mallet_path, corpus=None, num_topics=100, alpha=50, id2word=N
             raise ValueError("cannot compute LDA over an empty collection (no terms)")
         self.num_topics = num_topics
         self.topic_threshold = topic_threshold
-        self.alpha = alpha
+        if alpha is not None:
+            # Mallet's alpha parameter is the SumAlpha parameter
+            # (sum over topics of smoothing over doc-topic distributions. alpha_k = [SumAlpha] / [num topics])
+            # To keep the same interface with gensim, calculate the SumAlpha:
+            self.alpha = alpha * num_topics
```

Comment:
I'd prefer to do it the other way around: keep `alpha` gensim-style internally, and convert to SumAlpha only at the point of Mallet API = when calling Mallet.

I think it's safer wrt expectations of gensim users. Is there a good reason to do it the other way around?

Reply:
No, you are totally right. 
It is indeed safer for gensim users.
I addressed your feedback and substantially improved the documentation.